app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Constraints.scala

```diff
@@ -116,4 +116,10 @@ trait Constraints {
       case invalid => invalid
     }
   }
+
+  protected def checkCorrectLength(length: Int, invalidKey: String): Constraint[String] = Constraint {
+    case input if input.length != length =>
+      Invalid(invalidKey)
+    case _                               => Valid
+  }
```

Comment:
This is not necessary - there is already a `maxLength` constraint in `Constraints`

Reply:
Again, this is for exact length. Couldn'y gind an existing constraint.